Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
I think this should be as easy to set by the developer. The Hamburger should, as part of it's defining characteristic as a control, have an Accent Color. This accent color should affect the button background as well as the visual selection indicator (vertical rectangle on the left-bound margin of the hamburger items....not yet implemented). I envision this property to control this Accent Color that will affect at least two (maybe more visual elements). Foreground is universal for text color, Background is universal for the controls background, in the case of the Hamburger Background affects the background color for the Hamburger's Content area. We (rightfully) have a Pane Background. The only thing missing is the Accent Color (or whatever we want to name it) that will control the button / visual selection indicator's background color as seen in the image below:
![AccentColor](https://skycliffs.files.wordpress.com/2016/12/snip_20161211153115.png)